### PR TITLE
Flora data fixes

### PR DESCRIPTION
**Pull request by dcol2804**
Covering 2022 traits which are in the current release and 2023 traits which will be in the next version.

### CODE REVIEW

File: data/ABRS_2023/data.csv [hunk 1]
```diff
@@ -61518,7 +61516,7 @@ Flora_of_Australia,Cinnamomum baileyanum,leaf_length,lamina,50,,mm,minimum
 Flora_of_Australia,Cinnamomum camphora,leaf_length,lamina,40,,mm,minimum
 Flora_of_Australia,Cinnamomum laubatii,leaf_length,lamina,80,,mm,minimum
 Flora_of_Australia,Cinnamomum oliveri,leaf_length,lamina,85,,mm,minimum
-Flora_of_Australia,Cinnamomum propinquum,leaf_length,lamina,5,,mm,minimum
```
Comment (ehwenk) [inline data/ABRS_2023/data.csv:61521]: Mainly I've curious with how you caught this? By comparing to the same taxon in other studies?

File: data/NTH_2023/data.csv [hunk 2]
```diff
@@ -10938,11 +10938,7 @@ NT_flora,Pycnospora lutescens,leaf_shape,leaflet,filiform,filiform,,mode
 NT_flora,Calopogonium mucunoides,leaf_shape,leaflet,linear,"linear,",,mode
 NT_flora,Cullen patens,leaf_shape,leaflet,elliptical lanceolate,elliptic- lanceolate,,mode
 NT_flora,Galactia megalophylla,leaf_shape,leaves or leaflets,elliptical widely_lanceolate ovate,"elliptic, widelylanceolate, ovate cordate,",,mode
-NT_flora,Senna obtusifolia,leaf_shape,leaflets,terete,terete,,mode
```
Comment (ehwenk) [inline data/NTH_2023/data.csv:10941]: I'm wondering why you've removed all the leaflet shapes? Didn't we agree they could be included as long as we specified as a context that it was a measurement of a leaflet. Or am I missing something else

File: data/RBGV_2022/data.csv [hunk 3]
```diff
@@ -205,16 +205,16 @@ Aciphylla,Aciphylla simplicifolia," tufted herb 9-61 cm high, stems often purple
 Acronychia,Acronychia, small trees or shrubs. ,perennial,inferred_from_growth_form,,,shrub tree,shrubs trees,,,,,,,woody,
 Acronychia,Acronychia oblongifolia, glabrous shrub to medium-sized tree to circa 15 m high. ,perennial,inferred_from_growth_form,,,shrub tree,shrub tree,,,,,,,woody,
 Acrothamnus,Acrothamnus, shrubs. ,perennial,inferred_from_growth_form,,,shrub,shrubs,epiphyte,inferred_from_family,,,,,woody,
-Acrothamnus,Acrothamnus maccraei, erect to spreading shrub 1-3 m high; ,perennial,inferred_from_growth_form,,,shrub,shrub,epiphyte,inferred_from_family,,,erect spreading,erect spreading,woody,
```
Comment (ehwenk) [inline data/RBGV_2022/data.csv:208]: Are these instances where something was incorrectly inferred from genus?

File: data/WAH_2022_1/data.csv [hunk 4]
```diff
@@ -170,28 +170,28 @@ Centaurea,Centaurea melitensis, general biology. reproduction. reproduces by see
...
-Chordifex,Chordifex reseminans," rhizomes tufted, ",,,,,graminoid,,,,dioecious,,rhizomatous,rhizomes,herbaceous,inferred_from_growth_form
-Chordifex,Chordifex gracilior," rhizomes spreading, ",,,,,graminoid,,,,dioecious,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
+Chordifex,Chordifex reseminans," rhizomes tufted, ",,,,,graminoid,,,,,,rhizomatous,rhizomes,herbaceous,inferred_from_growth_form
+Chordifex,Chordifex gracilior," rhizomes spreading, ",,,,,graminoid,,,,,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
 Chordifex,Chordifex amblycoleus," rhizomes tufted, glabrous. dioecious (with separate male and female plants). ",,,,,graminoid,,,,dioecious,dioecious,rhizomatous,rhizomes,herbaceous,inferred_from_growth_form
-Chordifex,Chordifex chaunocoleus," rhizomes spreading, ",,,,,graminoid,,,,dioecious,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
-Chordifex,Chordifex isomorphus," rhizomes spreading, ",,,,,graminoid,,,,dioecious,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
```
Comment (ehwenk) [inline data/WAH_2022_1/data.csv:184]: Why was this removed

**Review by ehwenk — APPROVED** [review]
@dcol2804 I am happy to approve this provided I know that you've double, triple checked that the trait values you're removing are meant to be removed. I can't make judgements on every one or even look through them all.

File: data/NTH_2023/data.csv (hunk 2 above)
Comment (dcol2804) [reply data/NTH_2023/data.csv:10941]: I removed just a few leaflet shapes in the Senna genus because the description of the leaflet only included the petiole shape - terete. It didn't include any lamina shape

File: data/ABRS_2023/data.csv (hunk 1 above)
Comment (dcol2804) [reply data/ABRS_2023/data.csv:61521]: Yes, I got the average plant height from all species and then compared them with maximum leaf length - nearly all of the smallest leaves on tall trees were in a few families or genuses and these ones stood out

File: data/RBGV_2022/data.csv (hunk 3 above)
Comment (dcol2804) [reply data/RBGV_2022/data.csv:208]: That is correct

**Comment by ehwenk** [comment]
Thanks for all the answers. I figured if you could answer all of them easily, this was a good sampling of the possible changes and I'd trust the rest.

File: data/WAH_2022_1/data.csv (hunk 4 above)
Comment (dcol2804) [reply data/WAH_2022_1/data.csv:184]: Still investigating this one, I'd made the changes a long time ago so want to check the raw data one last time

Comment (dcol2804) [reply data/WAH_2022_1/data.csv:184]: Finished triple checking all the changes and I'm satisfied that they are all intentional and correct ;)

**Comment by dcol2804** [comment]
@ehwenk I've rechecked and double checked all the changes and satisfied that we can merge this in. Thanks for your prompt to go back and recheck!
